packages/modelviewer.dev/examples/staging-and-camera-control.html

```diff
@@ -299,7 +299,52 @@ <h4>Event listeners can cooperate with camera-controls</h4>
       </div>
     </div>
   </div>
-
+  <div class="sample">
+    <div id="demo-container-8" class="demo"></div>
+    <div class="content">
+      <div class="wrapper">
+        <div class="heading">
+          <h2 class="demo-title">Add other interactions like rotating environment map</h2>
+          <h4>Use parameters with rest turntable to rotate only the environment light</h4>
+          <p>By using shift key plus left click mouse or touch pad drag, one can rotate
+			  the environment map to see how the environment background would cast light
+			  onto the object from different angles when it is rotated. Updating orbit to 
+			  new theta value and then reseting the theta value to stop turntable from 
+		  rotating. </p>
+        </div>
+        <example-snippet stamp-to="demo-container-8" highlight-as="html">
+          <template>
+			  <model-viewer id="pan-demo" camera-controls src="../shared-assets/models/Astronaut.glb" environment-image="../assets/whipple_creek_regional_park_04_1k.hdr" 				alt="A 3D model of an astronaut"></model-viewer>
```

Comment:
This needs a unique id to ensure it doesn't conflict the other demos. Likewise, below you should select using `#your_id` so that you select the correct example.

Reply:
i see, ok let me make those changes.